primitive-types/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "primitive-types"
-version = "0.13.2"
```

Comment:
I don't understand this change, we still need a patch release for the [DecodeWith impls](https://github.com/paritytech/parity-common/pull/897/files#diff-644194b7e8fa32cd9256e7bc51dc80f24bce4494e3f2b767bb4156f39e4f5358R35)?